x-pack/plugins/observability_solution/observability_ai_assistant_app/public/functions/visualize_esql.tsx

```diff
@@ -60,6 +64,7 @@ interface VisualizeQueryResponsev1 {
   data: {
     columns: DatatableColumn[];
     userOverrides?: unknown;
+    errorMessages?: string[];
```

Comment:
should we remove this?

Reply:
Done 05398aceb2965f205104dd2dc30783b8c258702a